### PR TITLE
slight refactoring of code for clarity

### PR DESCRIPTION
**Pull request by moloko**

### CODE REVIEW

File: js/adapt-contrib-slider.js [hunk 1]
```diff
@@ -383,15 +383,15 @@ define([
...
+                this.$('.slider-scale-numbers').eq(0).css(
+                    this.model.get('_showScaleIndicator') ? {visibility: 'hidden'} : {display: 'none'}
+                );
             } else {
                 var $scaler = this.$('.slider-scaler');
                 var $markers = this.$('.slider-markers');
                 for (var i = 0, count = this.model.get('_items').length; i < count; i++) {
                     $markers.append("<div class='slider-line component-item-color'>");
-                    $('.slider-line', $markers).eq(i).css({left: this.mapIndexToPixels(i, $scaler) + 'px'});
+                    this.$('.slider-line', $markers).eq(i).css({left: this.mapIndexToPixels(i, $scaler) + 'px'});
```
Comment (moloko) [inline js/adapt-contrib-slider.js]: I thought the purpose of the comma in a JQuery selector was to 'select the combined results of all the specified selectors' 

i.e. `$('.slider-line', $markers)` would give you a JQuery object containing the current component's `.slider-markers` div plus **all** the `.slider-line` divs on the page.

Let me do a bit more digging.

Comment (oliverfoster) [inline js/adapt-contrib-slider.js]: The selector is the string which forms the first argument. The second argument, which denotes the context in which the selector is performed.

Comment (moloko) [inline js/adapt-contrib-slider.js]: ah, my bad - I was looking at [this](https://api.jquery.com/multiple-selector/)

I think the intent of 
```
$markers.find('.slider-line').eq(i).css({left: this.mapIndexToPixels(i, $scaler) + 'px'});
```
is much clearer, will change to that

**Comment by oliverfoster** [comment]
could you fix this line whilst you're here? https://github.com/adaptlearning/adapt-contrib-slider/blob/3f5ba1ac381a69930b538722f96255eba84b390d/js/adapt-contrib-slider.js#L280

it definitely needs a ``this.``

**Comment by moloko** [comment]
if you spot anything else let me know

I'll sort the funky indentation [here](https://github.com/adaptlearning/adapt-contrib-slider/blob/3f5ba1ac381a69930b538722f96255eba84b390d/js/adapt-contrib-slider.js#L401-L406)

File: js/adapt-contrib-slider.js [hunk 2]
```diff
@@ -383,26 +383,26 @@ define([
             this.$('.slider-markers').empty();
```
Comment (tomgreenfield) [inline js/adapt-contrib-slider.js]: While we're here could we convert this line to a variable declaration i.e.
```js
var $markers = this.$('.slider-markers').empty();
```
whilst replacing the occurence of `this.$('.slider-markers')` on line 385 and removing line 391?

**Pull request merged**